recipes/museotoolbox/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "museotoolbox" %}
+{% set version = "0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 30201a51d6033c1b191bd822741f7cbf073053926928d818ea874bfcad3f9917
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - gdal >=2
+    - numpy >=1.7
+    - pip
+    - python
+  run:
+    - gdal >=2
```

Comment:
is ``gdal`` a dependency required to run this project? or is it an optional dependency?
Because I didn't see it being defined on your setup.py and the metadata does not inform about it

Reply:
Or is it a dependency to run the tests?
if it is a dependency to test this project you can let it in a section inside of ``tests`` called ``requires``

Reply:
Hy @marcelotrevisani,
`gdal` is a dependency required to run this project, I updated the metadata of the develop branch to add this information, thank you.

Reply:
Thanks for the explanation! :)